CHANGELOG.md

```diff
@@ -27,6 +27,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Debug: Improve logic for halt locations used by breakpoints and stepping. (#1156)
 - Debug: Some in-scope variables are excluded from stack_trace. (#1156)
 - Fixed rtthost --scan-region to properly support memory range scannig. (#1192)
+- Debug: Poll RTT buffer data when target halts, and before each DAP request. (#1195)
+- Debug: Client only opens one RTT window if multiple RTT Channels are unnamed. (#1206)
```

Comment:
I think this is misleading. You put the description of the issue, not the solution. But the actual solution should be in the changelog :)

Reply:
Fair enough :) I've updated appropriately.